config/paths.js

```diff
@@ -35,4 +35,5 @@ module.exports = {
   appNodeModules: resolveApp('node_modules'),
   ownNodeModules: resolveApp('node_modules'),
   nodePaths: nodePaths,
+  novmc: resolveApp('node_modules/@novnc/novnc')
```

Comment:
Is this module need? Cause I didn't see in https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-3/react-console `@novnc/novnc` requirement.

Reply:
Ok, I found it, but it already in module dependencies and question is still relevant. Is this module need to install explicitly?

Reply:
@bond95 This module does not have in dist regular js code so it needs to me babelized.